utils.py

```diff
@@ -503,6 +504,37 @@ def convert_to_str(string_to_convert):
         return string_to_convert.encode('utf-8')
     return string_to_convert
 
+def get_sorted_nodes(nodes):
+    """Returns the nodes in topological sorted order.
+
+    Args:
+        nodes: list(dict). Unorderd list of nodes.
+
+    Returns:
+        list(dict). The list of nodes in topological sorted order.
+    """
+    unsorted_nodes = list(nodes)
+    orderd_skills_ids = set()
+    sorted_nodes = []
+    while len(unsorted_nodes):
+        for index, node in enumerate(unsorted_nodes):
+            prerequisite_skill_ids = set(node['prerequisite_skill_ids'])
+            acquired_skill_ids = set(node['acquired_skill_ids'])
+
+            if len(prerequisite_skill_ids) == 0 or (
```

Comment:
You don't need the first part of this, right? I.e. just `if prerequisite_skill_ids <= accumulated_skill_ids` should suffice?

Reply:
Yeah, that was by mistake. 
Done!